js/PaymentRequest/index.js

```diff
@@ -158,7 +152,7 @@ export default class PaymentRequest {
     // 6. If the displayItems member of details is present, then for each item in details.displayItems:
     validateDisplayItems(details.displayItems, ConstructorError);
 
-    // 7. Let selectedShippingOption be null.
+    // 7. Let selectedShippingOption and payment method be null.
```

Comment:
This comment should not change because this isn't where i initialized _paymentMethod